### DISCUSSION TITLE
Import problem : nested tags badly arranged, badly tagged notes

### DISCUSSION THREAD

**Discussion by mrtpcet** [standardnotes/forum#2875] [General]
MacOS 10.15.7
SN 3.23.96

I try to import a backup file into a new blank workspace and I have two major problems :

- some tags have disappeared from some notes
- so many nested tags were no longer stored under the right tag but ended up in root tags

**Comment by OA971** [comment]
I ran into a situation where I moved a Main tag into another Main Tag by Mistake. So it became a Sub Tag.
Now I want to move it back to be a Main Tag, but I can't seem to do it.
